README.md

```diff
@@ -21,9 +21,22 @@ Yargs helps you build interactive command line tools, by parsing arguments and g
 It gives you:
 
 * commands and (grouped) options (`my-program.js serve --port=5000`).
-* a dynamically generated help menu based on your arguments.
+* a dynamically generated help menu based on your arguments:
 
-> <img width="400" src="https://raw.githubusercontent.com/yargs/yargs/master/screen.png">
+```bash
```

Comment:
I think the `bash` syntax highlighting is less readable than no syntax highlighting, due to its random-seeming coloring of `&` and `return` in the help text.

````suggestion
```
````

Also, this was a problem in the previous version too, but I notice this whole code block should be indented with two spaces so it counts as being within the bullet point, rather than outside of the bulleted list. So ultimately this section of the README should be written like this:

````markdown
* a dynamically generated help menu based on your arguments:

  ```
  mocha [spec..]
  
  Run tests with Mocha
  
  Commands
    mocha inspect [spec..]  Run tests with Mocha                         [default]
    mocha init <path>       create a client-side Mocha setup at <path>
  
  Rules & Behavior
    --allow-uncaught           Allow uncaught errors to propagate        [boolean]
    --async-only, -A           Require all tests to use a callback (async) or
                               return a Promise                          [boolean]
  ```

* bash-completion shortcuts for commands and options.
````